### PR TITLE
Gv 3700 add new guides

### PR DESCRIPTION
**Pull request by Un-a**

### CODE REVIEW

File: packages/social-trader/src/components/center/center.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
   small?: boolean;
   middle?: boolean;
   large?: boolean;
+  justifyCenter?: boolean;
```
Comment (pavelzubov) [inline packages/social-trader/src/components/center/center.tsx:14]: Component closed for modification. Extend Center and change justify-content in extended component.

Comment (Un-a) [reply packages/social-trader/src/components/center/center.tsx]: Done

**Comment by pavelzubov** [comment]
The whole project doesn't require the JustifyContentSpaceAround and JustifyContentCenter components. Move it to the guides folder.

**Comment by Un-a** [comment]
@pavelzubov done

**Pull request merged**